irohad/torii/impl/query_service.cpp

```diff
@@ -88,4 +89,76 @@ namespace torii {
     return grpc::Status::OK;
   }
 
+  grpc::Status QueryService::FetchCommits(
+      grpc::ServerContext *context,
+      const iroha::protocol::BlocksQuery *request,
+      grpc::ServerWriter<iroha::protocol::BlockQueryResponse> *writer) {
+    log_->info("Fetching commits");
```

Comment:
These are rather debug statements, so `debug` can be used instead as it is done in `CommandService::StatusStream`. Same applies to logs below.